snow/context.go

```diff
@@ -95,6 +95,9 @@ type ConsensusContext struct {
 
 	// True iff this chain is currently state-syncing
 	StateSyncing utils.Atomic[bool]
+
+	// Subnet's stake this node must be connected to in order to report healthy.
+	MinPercentConnectedStakeHealthy float64
```

Comment:
I feel like this makes more sense inside the `Config`s than in the `Context` (as it doesn't need to be shared across consensus "gears" as alberto would say

Reply:
moved this to subnet configs